project/ReleaseUtils.scala

```diff
@@ -124,7 +121,8 @@ object ReleaseUtils {
        |
        |  depends_on "bash-completion"
        |  depends_on "coursier/formulas/coursier"
-       |  depends_on "openjdk@8+"
+       |  depends_on java: "1.8+" if MacOS.version < :big_sur
```

Comment:
Taken from https://github.com/coursier/homebrew-formulas/blob/master/coursier.rb

Reply:
Actually, it was pointed out to me that it's actually wrong.